src/format.ts

```diff
@@ -8,11 +8,11 @@ const numberFormat = memoize1<Intl.NumberFormat>(
 );
 const monthFormat = memoize1<Intl.DateTimeFormat>(
   (locale: string | string[] | undefined, month: "numeric" | "2-digit" | "long" | "short" | "narrow" | undefined) =>
-    new Intl.DateTimeFormat(locale, {timeZone: "UTC", month})
+    new Intl.DateTimeFormat(locale, {timeZone: "UTC", ...(month && {month})})
```

Comment:
Looks like an unrelated change? Or at least the context is not clear why this is part of this PR.

Reply:
I meant to mention this -- this was a necessary step after adding `"declaration": true,` to tsconfig.json. The type for `DateTimeFormatOptions` allows `month` to be optional, but not `month` to be present and set to `undefined`. So we need to only set month if it's defined. We hadn't run into this yet because this is a declaration emit errors, and those are produced as part of producing declaration files themselves and won't be encountered otherwise.